docker-cloud/cloud-swarm/register-swarms.md

```diff
@@ -0,0 +1,105 @@
+---
+description: how to register and unregister swarms in Docker Cloud
+keywords: register swarms, unregister swarms
+title: Register existing swarms
+---
+
+With Beta Swarm Mode, you can register existing
+swarms with Docker Cloud in order to easily
```

Comment:
s/in order to/to
s/a large number/many (or maybe "multiple") 

(this is what happens when you read FB comments on your day off. ;) )